### DISCUSSION TITLE
Where do I start

### DISCUSSION THREAD

**Discussion by m0rtalmike** [PPartisan/Roky#72] [General]
Would love to contribute but have no clue what I'm doing, never used GitHub before. If someone could give me pointers for first steps would love to get involved!

**Comment by KaiRaiChu** [comment]
Just eat an orange mate

**Comment by Robert-Davie** [comment]
https://play.kotlinlang.org/byExample
This is a useful tool suggested by Stefano for learning about Kotlin

**Reply by m0rtalmike** [reply]
Thank you Rob :)
